bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -136,6 +144,29 @@ <h2 class="go">{{ _('Download Firefox') }}</h2>
             <a class="fxos-link" href="{{ url('firefox.desktop.index') }}">{{ _('Learn more') }}</a>
           </div>
         </li>
+    {% if waffle.switch('mozorg-home-privacy-promo') and l10n_has_tag('privacy_day') %}
+      {% if waffle.switch('mozorg-home-privacy-promo-jan28') and LANG.startswith('en') %}
```

Comment:
I don't see why you need to nest these. The `mozorg-home-privacy-promo-jan28` switch shouldn't ever be active independent of the `mozorg-home-privacy-promo` switch, right? Also, you're checking both l10n_has_tag and LANG.startswith('en'), which is also redundant. I would just use the inner conditional and get rid of the outer one.
```